app/interactive_prompt.py

```diff
@@ -171,6 +172,44 @@ def noOp(data):
     def makeCommand(self, cmdLine, view):
         return {}, u'making stuff'
 
+    def openCommand(self, cmdLine, view):
+        """
+        Opens the file under cursor.
+
+        Symbols that can not be part of file names are in the `fileSeparators` 
+        set.  Potentially, all of them can be a part of the file but checking 
+        each combination on the disk would be expensive.
+        """
+        fileSeparators = set([' ', '<', '>', ':', ';', '\'', '"'])
+
+        penLine = view.textBuffer.lines[view.textBuffer.penRow]
+        wordStart = wordEnd = view.textBuffer.penCol
+
+        # Locate the boundaries of the word under the cursor
+        while wordStart > 0 and penLine[wordStart - 1] not in fileSeparators:
+            wordStart -= 1
+
+        penLineLength = len(penLine)
+
+        while wordEnd < penLineLength and penLine[wordEnd] != ' ':
```

Comment:
Did ya mean to change `penLine[wordEnd] != ' '` to
`penLine[wordEnd] not in fileSeparators`
so that it works similarly to line 189?